lib/modules/datasource/conan/index.ts

```diff
@@ -50,6 +55,38 @@ export class ConanDatasource extends Datasource {
     };
   }
 
+  @cache({
+    namespace: `datasource-${datasource}-revisions`,
+    key: ({ registryUrl, packageName }: GetReleasesConfig) =>
+      // TODO: types (#7154)
+      // eslint-disable-next-line @typescript-eslint/restrict-template-expressions
+      `${registryUrl}:${packageName}`,
+  })
+  override async getDigest(
+    { registryUrl, packageName, currentDigest }: DigestConfig,
+    newValue?: string
+  ): Promise<string | null> {
+    if (is.undefined(currentDigest) || is.undefined(registryUrl)) {
+      return null;
+    } else {
```

Comment:
```suggestion
    }
```

reduce nesting, the if block returns early, so no need for an else